### DISCUSSION TITLE
can you train dreambooth/lora via the ui or you just use the script?

### DISCUSSION THREAD

**Discussion by doodguymandood** [vladmandic/sdnext#322] [Q&A] [answered]
just curious as I see the embedding/hypernetwork in the train tab but nothing else

**Comment by vladmandic** [comment, marked as answer]
no ui support (yet), i plan to add it.

**Reply by doodguymandood** [reply]
ok awesome, a webui that can train all of those natively would be amazing

**Comment by vladmandic** [comment]
i have a fool-proof script all done, it hides 99% of complexities, just need a bit of quiet time to create ui. but given how this repo blew up in popularity last few days, that's difficult to find.

**Reply by 1r00t** [reply]
I'm looking forward to it. I struggled so much with the 1111 version. I'm confident it will be great!

**Reply by vladmandic** [reply]
yeah, but given how this repo has gained popularity, i'm struggling to find that "quiet time" - i wish some maintainers joined and other people started looking at reported issues or prs.

**Reply by 1r00t** [reply]
All I can say right now is thank you very much for all the time you put into this! I have some experience in python and web stuff but also not much free time.
